lib/MultiReporters.js

```diff
@@ -63,10 +63,21 @@ function MultiReporters(runner, options) {
         );
     }
 }
-util.inherits(MultiReporters, Base)
+
+util.inherits(MultiReporters, Base);
 
 MultiReporters.CONFIG_FILE = '../config.json';
 
+MultiReporters.prototype.done = function (failures, fn) {
+    if (_.size(this._reporters) === 0) return;
+
+    this._reporters.forEach(function(reporter) {
+        if (typeof reporter.done === 'function') {
+            reporter.done(failures, fn);
```

Comment:
Ah yes, you're spot on - Mocha expects the Reporter to invoke these arguments as `fn(failures)` after completing, so in order to support multiple reporters we need to:
a) ensure each reporter's `done()` function is invoked
b) call `fn(failures)` in our root level `done()` so that the process completes

I'll adapt my solution, thanks!